app/assets/stylesheets/forever_style_guide/modules/_list.scss

```diff
@@ -1,5 +1,34 @@
 /* list
 ---------------------------------------------------------------------------- */
+.list-group {
+  &.list-group-settings_list {
+    margin-bottom: 0;
+
+    .list-group-item {
+      border-radius: 0;
+      border: 0;
+      border-left: 5px solid transparent;
+      color: $color-forever_blue;
+      background-color: #f9f8f7;
+
+      &:hover {
+        color: $color-gray-600;
+        background-color: #e7e9e6;
+      }
+      &.active {
+        background-color: #e7e9e6;
```

Comment:
I think we should just use the existing style guide color palette. we can set the main bg as $color-gray-100 and the active and hover states as $color-gray-300